dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolProblem.java

```diff
@@ -23,8 +23,7 @@
 import javax.annotation.Nullable;
 
 /**
- * A missing or incompatible symbol. This constitutes the cause of a linkage error (without the
- * source class).
+ * A missing or incompatible symbol that causes a linkage error.
  *
  * @see <a
  *     href="https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/library-best-practices/glossary.md#linkage-error">
```

Comment:
This link is outdated. Would you update to https://jlbp.dev/glossary.html#linkage-error?

Reply:
done